modules/vault-cluster/main.tf

```diff
@@ -52,6 +52,18 @@ resource "aws_autoscaling_group" "autoscaling_group" {
     propagate_at_launch = true
   }
 
+  tag {
+    key                 = "using_dynamodb_backend"
+    value               = element(concat(aws_iam_role_policy.vault_dynamo.*.name, [""]), 0)
+    propagate_at_launch = true
+  }
+
+  tag {
+    key                 = "dynamodb_table_id"
+    value               = var.dynamodb_table_arn
+    propagate_at_launch = true
+  }
```

Comment:
Why set these as tags?

Reply:
Looking at the comment above these tags it looked like they were being used to force a wait before the cluster boots, If they aren't required then I can remove them.

https://github.com/hashicorp/terraform-aws-vault/blob/930df72d48a2db606b3f2e21f79c13912170a6da/modules/vault-cluster/main.tf#L33-L36

Reply:
Ah, gotcha. Could you add a similar comment for DynamoDB next to these tags?

Reply:
Ah, gotcha. Could you add a similar comment for DynamoDB next to these tags?